Source/Initialization/WarpXInitData.cpp

```diff
@@ -245,6 +245,96 @@ WarpX::PostProcessBaseGrids (BoxArray& ba0) const
         AMREX_D_TERM(},},})
         ba0 = BoxArray(std::move(bl));
     }
+
+    bool init_partition_species = false;
+    std::string density_function;
+    Real density_min=0;
+    Real efficiency_target = 0.7;
+    if (ParallelDescriptor::NProcs() > 1)
+    {
+        ParmParse pp_warpx("warpx");
+        std::string species;
+        pp_warpx.query("init_partition_species", species);
```

Comment:
We could remove this input parameter and instead always loop over all species.

Reply:
Instead we use a boolean flag `split_high_density_boxes`?

Reply:
`split_high_density_boxes_threshold` with a default of `1.1` (so that it does not split boxes if particles are uniformly distributed)